esrally/chart_generator.py

```diff
@@ -1503,6 +1590,18 @@ def generate_merge_time(chart_type, race_configs, environment):
 
     return structures
 
+def generate_ml_procesing_time(chart_type, race_configs, environment):
```

Comment:
* Nit (typo): `procesing` -> `processing`
* Nit: There should be two empty lines before each function (there are similar issues with the functions `generate_merge_count`, `generate_dashboard` and `generate_merge_time`)